### PR TITLE
smarter way of creating waypoints around a lump 

### PR DESCRIPTION
**Pull request by edith-langer**
taking the octomap into account

### CODE REVIEW

File: squirrel_perception_waypoint/CMakeLists.txt [hunk 1]
```diff
@@ -115,57 +115,7 @@ catkin_package(
 #   ${catkin_LIBRARIES}
 # )
 
-include_directories(include ${catkin_INCLUDE_DIRS})
+include_directories(include ${catkin_INCLUDE_DIRS} ${OCTOMAP_INCLUDE_DIRS})
 
 add_executable(squirrel_perception_waypoints_server src/squirrel_create_examine_waypoint.cpp)
-add_dependencies(squirrel_perception_waypoints_server squirrel_waypoint_msgs_generate_messages_cpp)
```
Comment (NadiaHG) [inline squirrel_perception_waypoint/CMakeLists.txt]: please do not remove this line!

**Review by ipa-nhg — COMMENTED** [review]
Travis is not happy, could you please check the  travis output https://travis-ci.org/squirrel-project/squirrel_perception/builds/332249313?utm_source=github_status&utm_medium=notification and fix the problem?

**Comment by edith-langer** [comment]
There seems to be a problem with the new v4r package. We are working on it...

**Comment by ipa-nhg** [comment]
just a comment. Buildbot is able to build the PR with the following v4r library version http://153.97.4.193/building/ubuntu/pool/main/v/v4r/

**Comment by ipa-nhg** [comment]
Travis use https://rwiki.acin.tuwien.ac.at/apt/v4r-release/pool/main/v/v4r/ , probably the last version

**Comment by edith-langer** [comment]
We removed the latest version of the v4r package. Could you please rerun the travis build?

File: squirrel_recognition/CMakeLists.txt [hunk 2]
```diff
@@ -40,6 +40,9 @@ include_directories(include
 )
 
 ## Declare a cpp executable
+add_executable(test_squirrel_recognizer src/test_recognizer.cpp ${incs})
```
Comment (NadiaHG) [inline squirrel_recognition/CMakeLists.txt]: CMake error: can not find this file

**Comment by ipa-nhg** [comment]
still not working :confused:  , please check the travis output!

**Comment by ipa-nhg** [comment]
please merge https://github.com/edith-langer/squirrel_perception/pull/1

File: squirrel_object_perception/src/squirrel_recognize_objects.cpp [hunk 3]
```diff
@@ -24,8 +23,10 @@
 #include "mongodb_store/message_store.h"
 #include <squirrel_object_perception_msgs/SegmentInit.h>
 #include <squirrel_object_perception_msgs/SegmentOnce.h>
+#include <squirrel_object_perception_msgs/BBox.h>
+#include <v4r_object_recognition_msgs/recognize.h>
```
Comment (NadiaHG) [inline squirrel_object_perception/src/squirrel_recognize_objects.cpp:27]:
```
fatal error: v4r_object_recognition_msgs/recognize.h: No such file or directory
 #include <v4r_object_recognition_msgs/recognize.h>
                                                   ^
```
missed dependency, where can be found this package?

Comment (edith-langer) [reply squirrel_object_perception/src/squirrel_recognize_objects.cpp:27]: They are part of our v4r ros wrappers. I changed CMakelists.txt and package.xml

**Comment by edith-langer** [comment]
Merging pull request 244 (squirrel_common) what you did right now should fix the problem.

**Comment by ipa-nhg** [comment]
Yes! Now is working I just had to update the buildbot release for squirrel_common!

MERGE!

**Pull request merged**